pkg/prototype/prototype.go

```diff
@@ -189,6 +200,25 @@ type ParamSchema struct {
 	Type        ParamType `json:"type"`
 }
 
+// IsRequired returns if this parameter is required. Parameters are required
+// if the default value is nil.
+func (ps *ParamSchema) IsRequired() bool {
+	return ps.Default == nil
+}
+
+func (ps *ParamSchema) QuotedValue(value string) (string, error) {
```

Comment:
I propose renaming this method `QuotedWithDefault` - as is the difference vs. `Quote` is not clear to the reader.